curricula/templates/curricula/partials/lesson_front.html

```diff
@@ -26,6 +26,15 @@ <h4>
 <div class="together overview-page row">
 
     <div class="col-sm-7 left-col">
+        {% comment %}
+        lessons using the CREATIVE_COMMONS_BY_NC_ND_IMAGE image also need
```

Comment:
should the contents of the comment be indented?

Reply:
yes they should!